### ISSUE TITLE
[Bug]: Multiple Player keyboard shortcuts arent't working during "SABR backoff" timer

### ISSUE THREAD

**Issue by Jack-McKalling** [FreeTubeApp/FreeTube#9101]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

1. Open any video
2. There is always a SABR timer below the video (4.8 seconds, but the .8 seem to be ignored)
3. During this timer, press the F button to go fullscreen
4. Notice that it doesn't go fullscreen
5. Before the timer finishes, click the button to go fullscreen
6. This does work

### Expected Behavior

When the button to go fullscreen is actually functional during this timer, the keybind to do the same should still respond as well.

### Issue Labels

accessibility issue, inconsistent behavior, keyboard control not working

### FreeTube Version

v0.24.0 Beta

### Operating System Version

Windows 10 Home 22H2

### Installation Method

Portable

### Primary API used

Local API

### Additional Information

Note this keybind works perfectly fine during video playback, and before the SABR implementation (since there wasn't any video player on screen yet during this timer to make fullscreen, I guess).

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by Amparose** [comment]
This also applies to (t) Theatre and (s) full-window option.

**Comment by Cyborgscode** [comment]
Suggestion: 

on keyboard-less devices like tablets, "f" is not an option at all. If the app is running in fullscreen, you will have issues ending it.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@Cyborgscode please elaborate some more what you exactly mean and how it related to this specific issue

**Comment by Cyborgscode** [comment]
Its just a general RFE, while you are on it. Think a little bit out of the Desktop box and add a solution for non desktop users as well. 

If my tablet would not have a home button and gestures to circumvent a missing home button, i would get stuck until i connect a keyboard / mouse to be able to reach the menu or could press "f"/F11 button. A general "toggle fullscreen mode" button somewhere in the ui, i.e. prefs section, would be helpfull in such a situation. 

The thing is, you wanne have the app in fullscreen, so the borders do not waste space. Even if you run a FullHD tabletscreen, you don't run it in 1:1 scale, you typically use 1:2 scales, so you can read texts better and have bigger buttons, as the tabletscreen is physically way smaller than a desktop one.

**Comment by Shadorc** [comment]
The issue comes from the fact that the method handling keyboard shortcuts (`keyboardShortcutHandler`) exits early if `!player || !hasLoaded.value`, and during SABR backoff timer, `hasLoaded` is false.

Keyboard shortcuts like fullscreen, full window, and theater mode are allowed via buttons, so they should work even if the video is not loaded.

I'm not confident about proposing a fix as I don't want to mess with the current order of `keyboardShortcutHandler` early exits, but these information could be helpful
